python/tvm/topi/transform.py

```diff
@@ -17,9 +17,12 @@
 # pylint: disable=invalid-name,consider-using-enumerate,redefined-outer-name
 """Injective transformation operators"""
 from __future__ import absolute_import as _abs
+import numpy as np
+from tables import Expr
```

Comment:
are these two lines necessary? i am getting a `tables` not found on my end and it seems like those two libs aren't referenced anyway

Reply:
I am hitting the following error:
```
E             0: tvm::codegen::CodeGenLLVM::CreateBufferPtr(llvm::Value*, tvm::runtime::DataType, llvm::ArrayRef<llvm::Value*>, tvm::runtime::DataType)
E                   at /home/yj/tvm/src/target/llvm/codegen_llvm.cc:737
E             File "/home/yj/tvm/src/target/llvm/codegen_llvm.cc", line 737
E           TVMError: 
E           ---------------------------------------------------------------
E           An error occurred during the execution of TVM.
E           For more information, please see: https://tvm.apache.org/docs/errors.html
E           ---------------------------------------------------------------
E             Check failed: (index->getType()->isIntegerTy()) is false: Expected buffer index to be an integer
```

Reply:
I think it's probably a spurious import (e.g. typed Expr and his IDE autoimported this)